Game/Src/TestEnemyMovement.gd

```diff
@@ -230,7 +231,7 @@ func spawn_player():
 
 func _on_timer_timeout():
 	var idx = rng.randi_range(0, 3)
-	GlobalVariables.glory -= 0.05
+	emit_signal("glory_increased",-gloryLoss)
```

Comment:
It would make sense to rename the signal to glory_changed and define the loss as negative

Reply:
There's a glory_changed as well, within glory_UI. It's kind of a mess of signals...